src/components/helpers.js

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../constants';
+
+/**
+ * webex Component Classes defines class names for components
+ *
+ * @param {object} props Data passed to the component
+ * @param {string} props.classBaseName  Custom CSS class to apply
+ * @param {string} props.className  Custom style to apply
+ * @returns {object} Classes
+ */
+
+export default function webexComponentClasses(classBaseName, className, otherClasses) {
```

Comment:
there are no props here, this function is not a component.

the signature should read:
```js
webexComponentClasses(componentBaseName, userClassName, otherClasses)
```

`otherClasses` is an object and must be listed in jsdoc.


I would advise building the complete class object like this:
```js
return classNames({
   [...]: true,
   [className]: !!className,
   ...otherClasses,
});